### PR TITLE
Use new USB device detection from ledgerjs

### PR DESCRIPTION
**Pull request by MortalKastor**
### Type

Performance improvement

### Context

LedgerHQ/ledgerjs#191

### Parts of the app affected

Device detection

### CODE REVIEW

File: src/helpers/deviceAccess.js [hunk 1]
```diff
@@ -39,7 +40,7 @@ export const withDevice: WithDevice = devicePath => job => {
     busy = true
     refreshBusyUIState()
     try {
-      const t = await TransportNodeHid.open(devicePath).catch(mapError)
+      const t = await retry(() => TransportNodeHid.open(devicePath), { maxRetry: 1 }).catch(mapError).catch(mapError)
```
Comment (meriadec) [inline src/helpers/deviceAccess.js]: is double catch intended?

Comment (MortalKastor) [reply src/helpers/deviceAccess.js]: Of course, it's safer.

**Comment by meriadec** [comment]
I think we'll need

```
apt-get install -y libudev-dev
```

on CI config (see [here](https://github.com/poanetwork/circleci-playground/blob/f8178278fe137db012eb67a6a18025c78f005385/.circleci/config.yml#L38-L41))

**Comment by meriadec** [comment]
Tested on Linux and I had to put `maxRetry: 2` in `withDevice` to avoid displaying the error (and stop pinging)

**Pull request merged**